hazelcast-client/src/test/java/com/hazelcast/client/cache/jsr/JsrClientTestUtil.java

```diff
@@ -31,22 +32,22 @@
     private JsrClientTestUtil() {
     }
 
-    static {
-        System.setProperty("javax.management.builder.initial", "com.hazelcast.cache.impl.TCKMBeanServerBuilder");
-        System.setProperty("org.jsr107.tck.management.agentId", "TCKMbeanServer");
-        System.setProperty(Cache.class.getName(), "com.hazelcast.cache.ICache");
-        System.setProperty(Cache.Entry.class.getCanonicalName(), "com.hazelcast.cache.impl.CacheEntry");
+    public static void setup() {
+        setSystemProperties("client");
```

Comment:
The JSR client tests already set the System properties in the code, so I centralized this in `JsrUtils.setSystemProperties()`.